monitor/flowgen/flowgen.go

```diff
@@ -102,25 +103,31 @@ func startWithBackends(
 // run runs a job exactly once. It returns false if the job was skipped.
 func run(ctx context.Context, job types.Job) (bool, error) {
 	log.Debug(ctx, "Flowgen: running job")
+	startTime := time.Now()
 
-	orderID, ok, err := job.OpenOrderFunc(ctx)
+	receipt, err := job.OpenOrderFunc(ctx)
 	if err != nil {
 		return false, errors.Wrap(err, "open order")
 	}
 
-	if !ok {
+	if !receipt.Success {
 		return false, nil
 	}
 
-	ctx = log.WithCtx(ctx, "order_id", orderID)
+	ctx = log.WithCtx(ctx, "order_id", receipt.OrderID)
 
 	log.Debug(ctx, "Flowgen: order opened")
 
-	if err := awaitClaimed(ctx, job, orderID); err != nil {
+	if err := awaitClaimed(ctx, job, receipt.OrderID); err != nil {
 		return false, errors.Wrap(err, "await claimed")
 	}
 
-	log.Info(ctx, "Flowgen: order claimed")
+	duration := time.Since(startTime)
+	log.Info(ctx, "Flowgen: order claimed",
+		"amount", bi.ToEtherF64(receipt.Expense.Amount),
+		"token", receipt.Expense.Token.Symbol,
+		"duration_minutes", duration.Minutes(),
```

Comment:
can just log `duration`, since the `.String()` method makes it nice and human readable.

Reply:
unless you plan to use it in graphs, then it cannot have a unit suffix.